clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -60,6 +61,7 @@ class AutoScalingWorker {
   private Boolean associatePublicIpAddress
   private String subnetType
   private Integer cooldown
+  private Collection<String> groupMetrics
```

Comment:
would be worth adding / updating something in AutoScalingWorkerUnitSpec around this field